### ISSUE TITLE
tailscaled 1.4.4 does not setup stun interface ip addresses

### ISSUE THREAD

**Issue by Bleistein** [tailscale/tailscale#1319]
We compiled version 1.4.4 of tailscaled unser macOS 11.2.1. We used the binary to replace version 1.2 in a working configuration.

https://login.tailscale.com/admin/machines reports a running daemon without any problem - but we are not able to ping/ssh to other machines in the network. The daemon is configured with utun5, ifconfig shows

utun5: flags=8051<UP,POINTOPOINT,RUNNING,MULTICAST> mtu 1280

So no ip addresses configured!

**Comment by mkramlich** [comment]
this is likely caused by #1201. if so, I'm working on a proper fix

**Comment by bradfitz** [comment]
Yes, dup of #1201. It's now fixed as of 88ab0173a7b46b5768fa907c0b3eb655f677c708.

@danderson, think we should backport it to 1.4.5?

**Comment by Bleistein** [comment]
I can confirm the new version fixed my problem...

**Comment by mkramlich** [comment]
@bradfitz @danderson if you do backport this into a 1.4.5 source tarball then I'll bump my brew #177 WIP up to use it. it might be nice if 1.4.5 also had 6680976b
